src/noflo-component-factory.js

```diff
@@ -75,27 +81,32 @@ function thenTrigger(fn, event /* arguments to fn and event */) {
  * Register listeners of the set of port definitions with this actual noflo Ports set.
  *
  * Usage:
- *  registerPorts({portName: new Port()}, {
+ *  registerPorts({portName: new Port()}, {portName: new PortFacade()}, {
```

Comment:
example needs to be updated to match the current 3 parameter API with facades as an array.